pkg/collector/corechecks/oracle-dbm/activity.go

```diff
@@ -455,75 +453,54 @@ func (c *Check) SampleSession() error {
 		obfuscate := true
 		var hasRealSQLText bool
 		if sample.Statement.Valid && sample.Statement.String != "" && !previousSQL {
-
 			// If we captured the statement, we are assigning the value
 			statement = sample.Statement.String
+			hasRealSQLText = true
+		} else if previousSQL && sample.PrevSQLFullText.Valid && sample.PrevSQLFullText.String != "" {
+			statement = sample.PrevSQLFullText.String
+			hasRealSQLText = true
+		} else if (sample.OpFlags & 8) == 8 {
+			statement = "LOG ON/LOG OFF"
+			obfuscate = false
+		} else if commandName != "" {
+			statement = commandName
+		} else if sessionType == "BACKGROUND" {
+			// The program name can contain an IP address, so we have to obfuscate it
```

Comment:
This comment doesn't make sense to me, we have to obfuscate an IP address?

Reply:
Oh interesting, I don't think we obfuscate that anywhere else in the product, e.g.: we send client IPs & the hostname for the database could obviously be an IP address. Is there a reason to treat this specific situation differently?

Reply:
> The host name alias usually resolves to a public address, which is different from the private IP address.

I mean that's dependent on how the customer has stuff set up, but I guess it's fine to obfuscate for now. My only concern is the text will be an IP address, but it will just show up as a single `?` I think? That seems confusing for showing in the UI. Should we have some kind of text represent the background process instead? @joelmarcotte recently did something similar for postgres background processes so I wonder if he has any thoughts on this.